clients/flash/air-client/src/org/bigbluebutton/air/main/views/TopToolbarMediatorAIR.as

```diff
@@ -15,7 +15,7 @@ package org.bigbluebutton.air.main.views {
 		}
 		
 		override protected function rightButtonClickHandler(e:MouseEvent):void {
-			
+			uiSession.pushPage(PageEnum.SETTINGS);

```

Comment:
You made all of the other toolbar mediators in the AIR client extend this class rather than the base class thereby making this class the new "base" for the AIR client specific code that the toolbars need. Using a base class directly can cause complications down the road and it's much safer to just extend the base class and make a new "main" toolbar mediator that extends this class does the exact same page transitions. Then you can remove the page transitions from here and remove the possible complications.